iaso/tests/api/test_completeness_stats.py

```diff
@@ -253,13 +256,14 @@ def test_base_row_listing(self):
                     },
                     "org_unit_type": {"name": "Country", "id": 1},
                     "parent_org_unit": None,
+                    "has_children": False,
                 },
             ],
             "has_next": False,
             "has_previous": False,
             "page": 1,
             "pages": 1,
-            "limit": 10,
+            "limit": "10",
```

Comment:
How is this a string ?

Reply:
the api was returning a string in fact ...